config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -1,27 +1,9 @@
 periodics:
-- name: periodic-cluster-api-provider-aws-test-creds
-  decorate: true
-  interval: 4h
-  labels:
-    preset-service-account: "true"
-    preset-aws-ssh: "true"
-    preset-aws-credential: "true"
-  extra_refs:
-  - org: kubernetes-sigs
-    repo: cluster-api-provider-aws
-    base_ref: master
-    path_alias: "sigs.k8s.io/cluster-api-provider-aws"
-  spec:
-    containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200128-8dae294-master
-      command:
-      - "./scripts/ci-aws-cred-test.sh"
-  annotations:
-    testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-aws
-    testgrid-tab-name: test-creds
 - name: periodic-cluster-api-provider-aws-e2e-master
   decorate: true
-  interval: 1h
+  decoration_config:
+    timeout: 4h
+  interval: 5h
```

Comment:
https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1501 will be forward porting the additional e2e tests we have from the `release-0.4` branch, which will require longer runtimes and intervals

Reply:
Just checked the conformance test job to see if this long of a timeout was necessary, but yeah, 4 hour timeout seems good 😬

Reply:
Everything is currently running serially, we can tweak in the future to run in parallel, which would help bring this down.